src/components/UserManagement/UserManagement.module.scss

```diff
@@ -1,8 +1,19 @@
+@import "styles/variablesscss";
 .pageContent {
   display: flex;
   flex-grow: 1;
+  flex-direction: column;
   font-family: "IBM Plex Sans", sans-serif;
 
+  .pageHeader {
```

Comment:
This ends up having a margin-right which makes it not span the full width of the page.

This seems to happen because the HistoryOrder component sets 
```
a {
  margin-right: var(--horizontal-spacing-normal);
}
```
without enclosing it in a class, so it applies globally.